package/network/ipv6/ds-lite/Makefile

```diff
@@ -36,7 +36,7 @@ endef
 
 define Package/ds-lite/install
 	$(INSTALL_DIR) $(1)/lib/netifd/proto
-	$(INSTALL_BIN) ./files/dslite.sh $(1)/lib/netifd/proto/dslite.sh
+	$(INSTALL_BIN) ./files/ipip6.sh $(1)/lib/netifd/proto/ipip6.sh
```

Comment:
This, and the renaming of dslite.sh to ipip6.sh, can go into a separated commit.
It's incomplete to only rename this script and keep all the other ds-lite names around.
You could either leave the file name unchanged, or rename the entire package to ipip6 and add a
```
PROVIDES:=ds-lite
```
line using a separated commit.